### DISCUSSION TITLE
Obtain the connectivity matrix

### DISCUSSION THREAD

**Discussion by CRISTIANJULIOCESAR** [lmcinnes/umap#779] [Q&A] [answered]
Hi! we are creating a method based totally on UMAP space but we use the increased or the lower-dimensional representation of the data by UMAP, we almost finish the paper. But we think that our method will render better results if we just used the connectivity matrix (weighted graph) that represents the high dimensionality of the data using your Riemannian metric. There is a function that returns this squared matrix?

Thank you in advance :)

**Comment by lmcinnes** [comment, marked as answer]
The ``graph_`` attribute of the fitted model contains this information.

**Reply by CRISTIANJULIOCESAR** [reply]
Thank you so much, it worked! We enjoy using UMAP, the best method for finding cell-similarities of scRNA-seq data so far.
